### DISCUSSION TITLE
Shortcut distillation column

### DISCUSSION THREAD

**Discussion by QingheGao** [IDAES/idaes-pse#1341] [General]
Hi,
I am a completely new user. I wonder if you guys have plans to integrate more distillation columns such as shortcut columns. Or there is already customed shortcut column implemetation I am not aware of.
Thanks in advance and thanks for your support!!

Best,
gao

**Comment by andrewlee94** [comment]
Hi @QingheGao. Unfortunately we do not have any plans currently to add more distillation columns into IDAES. The development team is very small and unfortunately we do not have the resources currently to look at distillation.

In case you haven't found them already, the models we currently have can be found here: https://github.com/IDAES/idaes-pse/tree/main/idaes/models_extra/column_models

We would like to expand our library, and if would gladly welcome contributions if you have the time to do so.

**Reply by QingheGao** [reply]
Thanks a lot for your quick response!! I will try to play it and see if I can contribute to it.
Thanks again!!
